keanu-project/src/main/java/io/improbable/keanu/tensor/Tensor.java

```diff
@@ -88,7 +88,11 @@ default boolean isScalar() {
     }
 
     default boolean isVector() {
-        return getRank() == 1;
+        if (getRank() == 2) {
```

Comment:
Can this be simplified to `getRank() == 2 && (getShape()[0] == 1 || getShape()[1] == 1)`?

Reply:
Good spot - I've updated it